panel/widgets/chat.py

```diff
@@ -204,8 +204,8 @@ class ChatEntry(CompositeWidget):
         Whether to display the timestamp of the message.
     """
 
-    value = param.ClassSelector(class_=(Viewable, str, int, float, _FileInputMessage), doc="""
-        The message contents. Can be a string, pane, widget, layout, etc.""")
+    value = param.Parameter(doc="""
```

Comment:
I want to be able to display any object that Panel can display. Not only a limited subset 😄

Reply:
If its Parameter most widgets input are broken that is why I changed it to limited subset

Reply:
I will try a workaround now by wrapping it in a bound function.

Reply:
I can see its already wrapped. The below is a minimum reproducible, test example 

```python
import panel as pn
import param

class Entry(pn.widgets.base.Reactive):
    value = param.Parameter()

entry = Entry(
    value=pn.widgets.Button(name="CLICK ME", button_type="primary"),
)
assert isinstance(entry.value, pn.widgets.Button)
```

Would you consider this a bug @philippjfr ?

Reply:
When debugging, I can see that there is an attribute `_ignored_refs` that can be set to `('value',)`. I believe that will solve the problem.

Reply:
It works. I'm closing this one.